src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.ts

```diff
@@ -103,6 +105,7 @@ export class DashboardV3Component extends PrometheusListHelper implements OnInit
     this.getPrometheusData(this.lastHourDateObject);
     this.getDetailsCardData();
     this.getTelemetryReport();
+    this.toggleAlertsWindow('critical');
```

Comment:
Rather than hard-coding this here, let's make it `critical` the default argument of this method.